colorfield/fields.py

```diff
@@ -53,12 +54,18 @@ def __init__(self, *args, **kwargs):
 
     def formfield(self, **kwargs):
         palette = []
-        if self.choices:
-            choices = self.get_choices(include_blank=False)
-            palette = [choice[0] for choice in choices]
+        if self.samples:
```

Comment:
The choices options has been removed, this is not backward-compatible at all.
We should keep both options: `choices` (restrictive) and `samples` (not restrictive).